### ISSUE TITLE
Injecting Already Translated Text into game

### ISSUE THREAD

**Issue by RagingBuracku** [Tsukihimates/Tsukihime-Translation#47]
I don't understand from the guide on how to mod tsukihime with the translation to actually insert what's already been translated, I got to the very end of the setup guide but I'm completely clueless on how to inject the translations into the game, cause I don't understand where the "upload" folder is supposed to be
![Screenshot 2021-10-05 002845](https://user-images.githubusercontent.com/59775668/135960168-c6dca487-a8d0-4961-b8de-91cfe7097a7b.png)

**Comment by VoidZ47** [comment]
If you have 1.0.2current_en_script downloaded, then you can paste that whole folder into the 'update' folder of deepluna, then run deepluna, open translation, and click insert, there will be a translated script text.mrg file with the date on it, then put that file in ryujinx mod folder and translated text should appear in the game, don't hesitate to ask again in case you can't figure it out ^^

**Comment by VoidZ47** [comment]
And don't forget to rename the  script_text(date) to just script_text.mrg when you paste it into the ryujinx mod folder

**Comment by RagingBuracku** [comment]
yeah I can't figure it out

**Comment by VoidZ47** [comment]
hmn, ill make a small tutorial video

**Comment by RagingBuracku** [comment]
that would be highly appreciated
much thanks

**Comment by VoidZ47** [comment]
oh wait i remembered that hakanaou made a tutorial video already, ill send the link to that

**Comment by VoidZ47** [comment]
well, that video is 30 minutes long....so not that...i'll figure something about for ya

**Comment by VoidZ47** [comment]
https://youtu.be/_SBfqHr9bHY 

here's a small video, i suppose you have deepluna set up already, and turn on subtitles for instructions

**Comment by RagingBuracku** [comment]
You think i could get that 30 minute, cause i think i dont have deepluna properly setup

**Comment by VoidZ47** [comment]
https://youtu.be/j8skThGYmDk

**Comment by RagingBuracku** [comment]
so is the movie thing that plays always gonna have JP text, cause I'm assuming that's a movie and nothing can be done about that cause I watched both videos, and I followed yours and I think it should work, I'm running it rn and sitting through the little movie bit

**Comment by VoidZ47** [comment]
Yeah, the prologue is unfortunately in movie format, other than that, you should have translated text

**Comment by waifwu** [comment]
@RagingBuracku The prologue video will be inserted into the game when we release the first patch, for now you can watch it form here https://youtu.be/X-v82yKkW8o

**Comment by RagingBuracku** [comment]
Sick, i also i tested last night and the translation works

**Comment by VoidZ47** [comment]
Great

**Comment by Hakanaou** [comment]
Does everything works now? If this is the case, it's better to close the issue.

**Comment by RagingBuracku** [comment]
Its pretty solid, so far just some small grammatical errors, but its definately understandable
